### ISSUE TITLE
Fails to parse statement with `union` keyword

### ISSUE THREAD

**Issue by byamba-ripple** [sbdchd/squawk#292]
Hello @sbdchd, `squawk` has been failing to parse simple select statements with `union` keyword. Please take a look at the details and let me know if there is anything wrong with I'm doing.

squawk version:
```bash
$ squawk --version
squawk 0.21.0
```

Error:
```bash
$ echo "SELECT a FROM b UNION SELECT x FROM y;" | squawk --verbose
18:56:42 [INFO] using config file path: .squawk.toml
18:56:42 [INFO] pg version: Some(Version { major: 11, minor: Some(9), patch: None })
18:56:42 [INFO] excluded rules: []
18:56:42 [INFO] assume in a transaction: false
18:56:42 [INFO] reading content from stdin
stdin:0:0: error: invalid-statement

   0 | SELECT a FROM b UNION SELECT x FROM y;

  note: Squawk schema failed to parse Postgres response. This indicates a bug with Squawk. Please report this error to https://github.com/sbdchd/squawk. Schema error: unknown variant `targetList`, expected `SelectStmt` at line 1 column 126
  help: Modify your Postgres statement to use valid syntax.

find detailed examples and solutions for each rule at https://squawkhq.com/docs/rules
```

**Comment by sbdchd** [comment]
Hmm I wonder if this is related to the recent parser update to Postgres’ 15 one

**Comment by chdsbd** [comment]
This bug was introduced with 98e1170dea408c19bcbbb2215d60ebd43d7dbf10

**Comment by chdsbd** [comment]
@byamba-ripple Should be fixed in the latest release. 0.22.0

**Comment by byamba-ripple** [comment]
Thank you for quickly fixing this, @chdsbd!
